tox/result.py

```diff
@@ -4,16 +4,15 @@
 
 import py
 
-from tox import __version__ as toxver
+import tox
 
 
 class ResultLog:
-
-    def __init__(self, dict=None):
-        if dict is None:
-            dict = {}
-        self.dict = dict
-        self.dict.update({"reportversion": "1", "toxversion": toxver})
+    def __init__(self, result_dict=None):
```

Comment:
do we have some pep-8 analyser to the project, how this gets through? (according to that really should be an  empty line here)

Reply:
having an empty line between class statement and first method, attribute declaration or docstring is optional and in fact most classes do not have that space in tox (48 vs 0 to be exact in main code - spaces are only in test classes). I prefer not having it also - I am a bit stingy with my vertical whitespace. I also avoid it inside methods and functions and rather split functionality out if for some reason I feel the need to have blocks)).

I will fix the classes in the tests still containing classes for consistency.